### DISCUSSION TITLE
How to submit LAMMPS jobs with several GPU crads?

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#1520] [Q&A]
May I ask you to use deepmd-kit installed with conda, and use lmp to run DPMD. If you want to use multiple cards in parallel, what is the command to submit the task (for example: lmp -sf gpu -pk gpu 2 -i input.lammps)? Thanks!

**Comment by njzjz** [comment]
You should try `mpirun -n 2 lmp -i input.lammps`.
